test/hydration-test.js

```diff
@@ -4,32 +4,44 @@ import UserRepo from '../src/User-repo';
 import User from '../src/User';
 
 describe('Hydration', () => {
+  let user 
+  let userData
   let hydrationData;
   let hydration;
 
   beforeEach(() => {
+    userData = {
+      id: 4,
+      name: 'Scooby Doo',
+      address: '123 Happy Lane',
+      email: 'Givemeajob@gmail.com',
+      strideLength: 4.3,     
+      dailyStepGoal: 10000,     
+      friends: [  16,  2, 6]    
+    }
+    user = new User(userData)
     hydrationData = [{
-      userID: 1,
+      userId: 1,
```

Comment:
We want this userId to be userID, as we want it to be like our data files that we are working with.